src/initialization/MOM_state_initialization.F90

```diff
@@ -1914,8 +1992,38 @@ subroutine initialize_sponges_file(G, GV, US, use_temperature, tv, param_file, L
 
 
 
-  endif
+ endif
+
+  ! The remaining calls to set_up_sponge_field can be in any order.
+!  if ( use_temperature .and. .not. time_space_interp_sponge) then
+!    call MOM_read_data(filename, potemp_var, tmp(:,:,:), G%Domain)
+!    call set_up_sponge_field(tmp, tv%T, G, GV, nz, Layer_CSp)
+!    call MOM_read_data(filename, salin_var, tmp(:,:,:), G%Domain)
+!    call set_up_sponge_field(tmp, tv%S, G, GV, nz, Layer_CSp)
+!  elseif (use_temperature) then
+!    call set_up_ALE_sponge_field(filename, potemp_var, Time, G, GV, US, tv%T, ALE_CSp)
+ !    call set_up_ALE_sponge_field(filename, salin_var, Time, G, GV, US, tv%S, ALE_CSp)
+
+ if (sponge_uv .and. .not. use_ALE) call MOM_error(FATAL,'initialize_sponges_file: '// &
+                       'UV damping to target values only available in ALE mode')
+
+ if (sponge_uv .and. use_ALE)  then
+
+   filename = trim(inputdir)//trim(state_uv_file)
+   call log_param(param_file, mdl, "INPUTDIR/SPONGE_STATE_UV_FILE", filename)
+   if (.not.file_exists(filename, G%Domain)) &
+     call MOM_error(FATAL, " initialize_sponges: Unable to open "//trim(filename))
+
+   call set_up_ALE_sponge_vel_field(filename, u_var, filename, v_var, Time, G, GV, US, ALE_CSp, u, v)
```

Comment:
Does this need to handle the case of fixed velocity fields as well? This block initialises the velocity fields as varying, regardless of `time_space_interp_sponge`

Reply:
Should work provided a time axis exists in the file. To be safe, I added a call to initialize using the fixed interface.

Reply:
I think that we should accept this commit as it currently stands, but if this concern is manifest in practice, it can be revisited by introducing either a new PR to correct it or a new issue. As of now, this PR is passing all of our tests.